connexion/app.py

```diff
@@ -87,7 +87,8 @@ def common_error_handler(exception):
         return problem(title=exception.name, detail=exception.description, status=exception.code)
 
     def add_api(self, swagger_file, base_path=None, arguments=None, auth_all_paths=None, swagger_ui=None,
-                swagger_path=None, swagger_url=None, validate_responses=False, resolver=Resolver()):
+                swagger_path=None, swagger_url=None, validate_responses=False, resolver=Resolver(),
+                security_decorator=None):
```

Comment:
Probably this option should be added to the constructor of `App` class too, in case you want to add this security decorator globally in your APIs.